packages/admin/cms-admin/src/contentScope/ContentScopeSelect.tsx

```diff
@@ -0,0 +1,177 @@
+                                    startAdornment={
+                                        <InputAdornment position="start">
+                                            <Search />
+                                        </InputAdornment>
+                                    }
+                                    placeholder={intl.formatMessage({
+                                        id: "contentScopeSelect.searchInput.placeholder",
+                                        defaultMessage: "Search...",
+                                    })}
+                                    value={searchValue}
+                                    onChange={(event) => setSearchValue(event.currentTarget.value)}
+                                    endAdornment={
+                                        <ClearInputAdornment
+                                            onClick={() => setSearchValue("")}
+                                            hasClearableContent={searchValue !== ""}
+                                            position="end"
+                                        />
+                                    }
+                                    autoFocus
+                                    fullWidth
+                                />
+                            </Box>
+                            <Divider />
+                        </>
+                    )}
+                    <List>
+                        {groups.map((group, index) => {
+                            const showGroupHeader = hasMultipleDimensions;
+                            const showGroupDivider = showGroupHeader && index !== groups.length - 1;
+
+                            return (
+                                <React.Fragment key={group.value}>
+                                    {showGroupHeader && <ListSubheader>{humanReadableLabel(group)}</ListSubheader>}
+                                    {group.options.map((option) => (
+                                        <ListItemButton
+                                            key={JSON.stringify(option)}
+                                            onClick={() => {
+                                                hideDropdown();
+                                                onChange(optionToValue<Value>(option));
+                                                setSearchValue("");
+                                            }}
+                                            selected={option === selectedOption}
+                                        >
+                                            {renderOption?.(option)}
+                                        </ListItemButton>
+                                    ))}
+                                    {showGroupDivider && <Divider sx={{ margin: 2, borderColor: "grey.50" }} />}
+                                </React.Fragment>
+                            );
+                        })}
+                    </List>
```

Comment:
There should probably be some kind of "No results found" text, for when there are no `groups`. 

<img width="263" alt="image" src="https://github.com/vivid-planet/comet/assets/6264317/89eed2f6-1e3d-456e-8d98-642a6d64a877">

Reply:
c25fc8ad6e0cf87faeccabfda20f11d7026b9c85